jest-setup.ts

```diff
@@ -3,32 +3,45 @@ import * as Adapter from 'enzyme-adapter-react-16';
 import { noop } from 'lodash';
 
 const adapter = Adapter as any;
+
 configure({ adapter: new adapter.default() });
 
 // https://github.com/FezVrasta/popper.js/issues/478
 if (window.document) {
-    document.createRange = () => ({
-        setStart: noop,
-        setEnd: noop,
-        commonAncestorContainer: {
-            nodeName: 'BODY',
-            ownerDocument: document,
-        },
-    } as Range);
+    document.createRange = () =>
+        ({
+            setStart: noop,
+            setEnd: noop,
+            commonAncestorContainer: {
+                nodeName: 'BODY',
+                ownerDocument: document,
+            },
+        } as Range);
 }
 
-window.matchMedia = jest.fn(() => ({
-    matches: false,
-    addListener: noop,
-    addEventListener: noop,
-    removeListener: noop,
-    removeEventListener: noop,
-} as MediaQueryList));
+window.matchMedia = jest.fn(
+    () =>
+        ({
+            matches: false,
+            addListener: noop,
+            addEventListener: noop,
+            removeListener: noop,
+            removeEventListener: noop,
+        } as MediaQueryList),
+);
 
-Object.defineProperty(window.navigator, 'userAgent', (value => ({
-    get() { return value; },
-    set(v) { value = v; },
-}))(window.navigator.userAgent));
+Object.defineProperty(
+    window.navigator,
+    'userAgent',
+    ((value) => ({
+        get() {
+            return value;
+        },
+        set(v) {
+            value = v;
+        },
+    }))(window.navigator.userAgent),
+);
```

Comment:
This file appears to only have formatting changes.
Could you please confirm that it is all done by code rather than manual editing so that it does not change in the future?

Reply:
This seems to be done by auto saving.